fix(api-service): remove ff flag for translation

scopsy opened the pull request:
### What changed? Why was the change needed?

<!-- Also include any relevant links, such as Linear tickets, Slack discussions, or design documents. -->

### Screenshots

<!-- If the changes are visual, include screenshots or screencasts. -->

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Related enterprise PR

<!-- A link to a dependent pull request  -->

### Special notes for your reviewer

<!-- Specific instructions or considerations you want to highlight for the reviewer. -->

</details>

scopsy committed:
Remove translation feature flag check and update timeout

Eliminated the translation feature flag check from BaseTranslationRendererUsecase, enforcing workflow ID requirement with error logging and exception. Increased internal API request timeout from 30s to 60s in ExecuteBridgeRequest for improved reliability.

scopsy committed:
Update base-translation-renderer.usecase.ts

scopsy committed:
Bypass translation for non-enterprise environments

Added checks to skip translation logic in BaseTranslationRendererUsecase when NOVU_ENTERPRISE is not set to 'true'. This ensures translation is only executed in enterprise environments.

scopsy committed:
Update email-output-renderer.usecase.ts

scopsy committed:
Add dbWorkflow parameter to translation renderer usecases

Introduces the dbWorkflow parameter to methods in BaseTranslationRendererUsecase and PushOutputRendererUsecase, allowing workflow entity data to be passed for translation rendering. This change supports enhanced context for translation operations.

ChmaraX commented on apps/api/src/app/environments-v1/usecases/output-renderers/base-translation-renderer.usecase.ts:
```diff
@@ -99,7 +86,14 @@ export abstract class BaseTranslationRendererUsecase {
     locale?: string;
   }): Promise<string | Record<string, unknown>> {
     if (!workflowId) {
```
I think this `if` statement was added during layouts impl. - I would suggest verifying if it doesn't break anything in layouts.

scopsy committed:
Add organization context to workflow and renderers

Pass organization entity through workflow construction and output renderer usecases to support organization-specific logic, including translation and rendering. Updates method signatures and command objects to include organization where relevant.

ChmaraX committed:
fix(translations): early errors, no fallbacks, better logging

ChmaraX committed:
fix test

ChmaraX committed:
submodule sync
